templates/gitlab.rb.j2

```diff
@@ -68,6 +68,11 @@ gitlab_rails['ldap_servers'] = YAML.load <<-EOS
     user_filter: '{{ values.user_filter | default(gitlab_ldap_user_filter) }}'
 
     attributes: {{ values.attributes | default(gitlab_ldap_attributes) }}
+{% if gitlab_version is defined and gitlab_version == "gitlab-ee" %}
```

Comment:
I think if this was `gitlab_edition` instead of `gitlab_version` it would work.

Reply:
ah yes truly! I skipped into the wrong line in the README.
Silly me, fixing it now.